cmd/zdb/zdb.c

```diff
@@ -745,6 +745,9 @@ dump_metaslab_stats(metaslab_t *msp)
 	dump_histogram(rt->rt_histogram, RANGE_TREE_HISTOGRAM_SIZE, 0);
```

Comment:
If you are going to add a copyright line in other places, maybe add one to zdb.c as well?